recipes/SimpleElastix/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Tell the build to use conda's Python
+SET PYTHON_EXECUTABLE=$PYTHON
+SET PYTHON_INCLUDE_DIR=$PREFIX/include
+SET PYTHON_LIBRARY=$PREFIX/lib
+
+# Build
+mkdir build
+cd build
+cmake ../SimpleElastix/SuperBuild
+make -j4
```

Comment:
instead of `4` I guess that you should use `$CPU_COUNT`. (Is that correct @jakirkham?)